metadata.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "simp-incron",
-  "version": "0.0.3",
+  "version": "0.0.4",
```

Comment:
I believe this should be a minor bump, since there is new functionality in the top-level API